doc/kb_sdk_complete_module_info.md

```diff
@@ -107,6 +109,22 @@ publications :
     	link: http://www.usadellab.org/cms/?page=trimmomatic
 ```
 
+##### Screenshots
+You can add screenshots (or other relevant images) to the "img/" folder in the same fashion as the icon image.  These screenshots should be configured in the **display.yaml** file as a list with hyphens for each element, such as
+
+```
+screenshots:
+    - screenshot_1.png
+    - screenshot_2.png
+```
+
+If you do not want to have any screenshots, leave the **screenshots:** list configuration blank
+
+```
+screenshots: []
+```
+
+##### Example
 For an example of a complete App Info page that would be acceptable for public deployment, please see:
 
     https://appdev.kbase.us/#appcatalog/app/kb_trimmomatic/run_trimmomatic/dev
```

Comment:
Dylan, the Trimmomatic app info page looks good, except there are some formatting problems with the links in the references section at the end.  
Related publications
'Bolger AM, Lohse M, Usadel B., (2014) Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics. 2014 Aug 1;30(15):2114-20. doi: 10.1093/bioinformatics/btu170.' http://www.ncbi.nlm.nih.gov/pubmed/24695404
http://www.usadellab.org/cms/?page=trimmomatichttp://www.usadellab.org/cms/?page=trimmomatic